src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -740,6 +742,7 @@ function ExpandedPostDetails({
   const {_, i18n} = useLingui()
   const openLink = useOpenLink()
   const isRootPost = !('reply' in post.record)
+  const niceDate = isAndroid ? niceDateAndroid : niceDateWebOriOS
```

Comment:
We shouldn't need this. The build system will automatically pick up the `.android` file fork; you should import the main file like before.

Reply:
Sorry I didn't notice that before, revented `PostThreadItem.tsx`.